### PR TITLE
Add php8.0

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@phil-davis plz review - THX

File: v8.0/manifest.tmpl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+image: owncloudci/php:7.4
```
Comment (phil-davis) [inline v8.0/manifest.tmpl:1]: s/7.4/8.0/

in this file

File: v8.0/overlay-amd64/etc/php/7.4/apache2/conf.d/20-oci8.ini [hunk 2]
```diff
@@ -0,0 +1 @@
+../../mods-available/oci8.ini
```
Comment (phil-davis) [inline v8.0/overlay-amd64/etc/php/7.4/apache2/conf.d/20-oci8.ini:1]: The folder path has `7.4`  in it - rename that folder to 8.0 ?

File: v8.0/overlay/etc/php/7.4/apache2/conf.d/10-opcache-custom.ini [hunk 3]
```diff
@@ -0,0 +1 @@
+opcache.revalidate_freq=0
```
Comment (phil-davis) [inline v8.0/overlay/etc/php/7.4/apache2/conf.d/10-opcache-custom.ini:1]: `7.4` is in the folder path. Rename to `8.0` ?

**Review by phil-davis — APPROVED** [review]
LGTM - we will really know when we use it in CI

**Comment by DeepDiver1975** [comment]
> LGTM - we will really know when we use it in CI

plz merge if you feel like it ... I have no access to this repo/orga - THX

**Comment by phil-davis** [comment]
This will be very good to have. PHP 8.0 release schedule https://wiki.php.net/todo/php80
RC4 should be released in a few days, and general release around 2020-11-26

`sabre/dav` already has CI passing with PHP 8.0, so WebDAV should "just work". We will see what other PHP dependencies still need to get PHP 8.0 support.

**Pull request merged**